metrics_utility/automation_controller_billing/report/report_ccsp_v2.py

```diff
@@ -8,9 +8,18 @@
 from openpyxl.styles import Alignment, Border, Font, PatternFill, Side
 from openpyxl.utils import get_column_letter
 from openpyxl.utils.dataframe import dataframe_to_rows
-
+import os
 from metrics_utility.automation_controller_billing.report.base import Base
 
+DIRECT = 0
+INDIRECT = 1
+# EDGE = 2
+
+def get_optional_collectors():
+    return os.environ.get('METRICS_UTILITY_OPTIONAL_COLLECTORS', 'main_jobevent').split(",")
+
+environs = get_optional_collectors()
+INCLUDE_INDIRECT = ('indirect_nodes' in environs)
```

Comment:
similar try to pass all env vars via https://github.com/ansible/metrics-utility/blob/devel/metrics_utility/management/commands/build_report.py#L179

so we don't handle this in each individual file

Reply:
Yeah, I know, I had in in helpers, but import was not working, so I wanted to solve this in morning, so I have pushed my code.